packages/react-vapor/src/components/color-picker/ColorPicker.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {ChromePicker, ChromePickerProps, ColorResult} from 'react-color';
+import {connect} from 'react-redux';
+import {isString, noop, uniqueId} from 'underscore';
+import {IReactVaporState} from '../../ReactVapor';
+import {IDispatch} from '../../utils/ReduxUtils';
+import {addInput, changeInputValue, removeInput} from '../input/InputActions';
+import {InputConnected} from '../input/InputConnected';
+import {InputSelectors} from '../input/InputSelectors';
+
+export interface IColorPickerProps extends ChromePickerProps {
+    id?: string;
+    defaultColor?: string;
+}
+
+export interface IColorPickerDispatchProps {
```

Comment:
Remove this type, instead we use `ReturnType<typeof mapDispatchToProps>`. E.g,

`export class ColorPicker extends React.Component<IColorPickerProps & ReturnType<typeof mapStateToProps> & ReturnType<typeof mapDispatchToProps>> {`